Memorize/EmojiMemoryGameView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  EmojiMemoryGameView.swift
+//  Memorize
+//
+//  Created by Zafer CELALOGLU on 10.02.2025.
+//
+
+import SwiftUI
+
+struct EmojiMemoryGameView: View {
+    @ObservedObject var viewModel = EmojiMemoryGame()
```

Comment:
You should change that to StateObject. Use ObservedObject only when passing an object through the initializer from outside of the view.
Or, if you want to use an ObservedObject, do not initialize it:
```swift
@ObservedObject var viewModel: EmojiMemoryGame
```